.github/workflows/publish.sh

```diff
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think this will cause PRs to clobber our `gh-pages` branch. My guess (Actions docs are unclear) is that merging a PR will fire the `push` event?
```suggestion
```